dvc/repo/get.py

```diff
@@ -33,28 +45,33 @@ def get(url, path, out=None, rev=None):
     tmp_dir = os.path.join(dpath, "." + str(shortuuid.uuid()))
     try:
         with external_repo(cache_dir=tmp_dir, url=url, rev=rev) as repo:
-            # Note: we need to replace state, because in case of getting DVC
-            # dependency on CIFS or NFS filesystems, sqlite-based state
-            # will be unable to obtain lock
-            repo.state = StateNoop()
+            full_path = os.path.join(repo.root_dir, path)
+            path_in_repo = path_isin(full_path, repo.root_dir)
+            if os.path.exists(full_path) and path_in_repo:
+                _copy_git_file(repo, path, out)
```

Comment:
You can simply return here, so that you don't have to put all of that bottom code under the `else` 🙂